package.json

```diff
@@ -42,6 +42,9 @@
     "graceful-fs": "4.1.4",
     "jasmine-core": "2.4.1",
     "jquery": "2.2.3",
+    "angular-ui-date": "1.0.1",
+    "angular-ui-slider": "0.3.2",
+    "jquery-ui": "git+https://github.com/jquery/jquery-ui.git#1.11.4",
```

Comment:
`jquery/jquery-ui.git#1.11.4` is the github syntax; that's what we use for openlayers